install/pre-install-payload/containerd/payload.sh

```diff
@@ -22,10 +22,12 @@ function setup_env_for_arch() {
 		"linux/amd64") 
 			image="docker.io/library/centos"
 			kernel_arch="x86_64"
+			golang_arch="amd64"
 			;;
 		"linux/s390x")
 			image="docker.io/library/clefos"
 			kernel_arch="s390x"
+			golang_arch="amd64"
```

Comment:
Hi, should this be `golang_arch="s390x"`

Reply:
Oops, fixed, thanks @hbrueckner!

Reply:
Thanks